topics/variant-analysis/tutorials/beacon_cnv_query/tutorial.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: tutorial_hands_on
+
+title: "Querying a Beacon Database for Copy Number Variants (CNVs)"
+zenodo_link: 'https://zenodo.org/records/10658688'
+subtopic: 'data-management'
+priority: 1
+questions:
+- What does the term "Beacon" refer to in the context of genomic databases?
+- How can the Beacon2 CNV tool be utilized to verify the presence of specific Copy Number Variants (CNVs)?
+objectives:
+- Comprehend the concept of Beacon databases and their role in genomic research.
+- Demonstrate proficiency in querying Beacon databases to identify and analyze specific Copy Number Variants (CNVs).
+time_estimation: 30M
+key_points:
+- Understanding Beacon Databases involves grasping their purpose and significance in genomic research and data sharing.
+- Mastering the Beacon2 CNV Tool includes developing skills to effectively query and analyze specific CNVs within the Beacon2 database.
+- Efficient Querying for CNVs focuses on accurate data retrieval and precise query formulation using the Beacon2 tool.
+- Hands-On Application provides practical experience in querying Beacons, ensuring data integrity, and troubleshooting queries.
+- Recognizing the Importance of CNVs highlights the role of CNVs in genetic variation and their relevance in research using Beacon databases.
+contributors:
+- khaled196
+- poterlowicz-lab
+---
```

Comment:
you can also define the other tutorial as a follow-up training here, then it will be recommended at the bottom of the tutorial, with syntax like:

```
follow_up_training:
- type: internal
  topic_name: statistics
  tutorials:
      - age-prediction-with-ml
```

(see also https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html#follow_up_training)